src/components/hint/index.tsx

```diff
@@ -172,7 +217,7 @@ class Hint extends BaseComponent {
   }
 
   get containerWidth() {
-    const {containerWidth} = this.getThemeProps();
+    const {containerWidth} = getThemeProps(this.props, this.context);
```

Comment:
When wrapping the component with `asBaseComponent` you can simply extract from `this.props`, as the themeProps already passed to the WrappedComponent (see asBaseComponent line #54).

Reply:
I see, will add the change 👍🏼